src/image/ImagePointThread.js

```diff
@@ -1,17 +1,31 @@
 import AnnotationThread from '../AnnotationThread';
 import { showElement, shouldDisplayMobileUI, repositionCaret, findElement } from '../util';
 import { getBrowserCoordinatesFromLocation } from './imageUtil';
-import { STATES } from '../constants';
-
-const POINT_ANNOTATION_ICON_HEIGHT = 31;
-const POINT_ANNOTATION_ICON_DOT_HEIGHT = 8;
-const POINT_ANNOTATION_ICON_WIDTH = 24;
+import {
+    STATES,
+    ANNOTATION_POPOVER_CARET_HEIGHT,
+    POINT_ANNOTATION_ICON_WIDTH,
+    POINT_ANNOTATION_ICON_HEIGHT,
+    POINT_ANNOTATION_ICON_DOT_HEIGHT,
+    SELECTOR_CLASS_ANNOTATION_POPOVER
+} from '../constants';
 
 class ImagePointThread extends AnnotationThread {
     //--------------------------------------------------------------------------
     // Abstract Implementations
     //--------------------------------------------------------------------------
 
+    /**
+     * Gets the popover parent for image point threads. The popover parent
+     * should not the image element but rather the annotatedElement
+     *
+     * @override
+     * @return {HTMLElement} The correct parent based on mobile view or not
+     */
+    getPopoverParent() {
+        return shouldDisplayMobileUI(this.container) ? this.container : this.annotatedElement;
```

Comment:
Is this still the case w/ multi-page image files w/ annotations?

Reply:
yeah, looks like this is needed for all images